src/playlist-maker.js

```diff
@@ -3,6 +3,16 @@ var setupAutoadvance = require('./autoadvance.js');
 var isArray = Array.isArray || function(array) {
   return Object.prototype.toString.call(array) === '[object Array]';
 };
+var isInSources = function(arr, src) {
+  var i = 0;
+  for (; i < arr.length; i++) {
+    if (arr[i] === src || arr.src === src) {
```

Comment:
it takes an array and a source string, like what is returned from `player.currentSrc()`.
Though, I think I may have messed it up in the conversion from the other plugin.